docs-src/docs/guides/walkthrough/intro.md

```diff
@@ -0,0 +1,24 @@
+# An Iterative Walkthrough
+
+This guide aims to take you through three increasingly-complex deployment examples using Layer0.
+Successive sections build upon the previous ones, and each deployment can be completed either through the Layer0 CLI directly, or through Terraform using our custom [Layer0 Terraform Provider](/reference/terraform-plugin).
```

Comment:
I believe this is the first time we mention terraform in our docs, if following them linearly. I'm thinking we should add a hyperlink here to terraform's docs and add some details for terraform in a section like `/setup/install/`